Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -202,10 +191,14 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	void OnTakeActorsFromPoolCompleted(const TArray<FPoolObjectData>& CreatedObjects);
 
 	/** Triggers when a spot is loaded */
-	UFUNCTION(BlueprintNativeEvent, Blueprintable, Category="C++", meta=(BlueprintProtected))
+	UFUNCTION(BlueprintNativeEvent, BlueprintCallable, Category="C++", meta=(BlueprintProtected))
 	void OnSpotComponentLoad(class UPSSpotComponent* SpotComponent);
 
 	/** Is called from AsyncLoadGameFromSlot once Save Game is loaded, or null if it failed to load. */
 	UFUNCTION(BlueprintNativeEvent, BlueprintCallable, Category = "C++", meta = (BlueprintProtected))
 	void OnAsyncLoadGameFromSlotCompleted(const FString& SlotName, int32 UserIndex, class USaveGame* SaveGame);
+
+	/** Is called to update the stars actors and in widgets when finish to save date in save file */
+	UFUNCTION(BlueprintNativeEvent, BlueprintCallable, Category = "C++", meta = (BlueprintProtected))
```

Comment:
`UpdateProgressionUI` is not event, therefore no need for blueprint developers override this function, so please remove `BlueprintNativeEvent`

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/40/commits/4048173ca1e9b5e7f2f0542ac5d0110103ee864d